content/jobs.md

```diff
@@ -5,35 +5,54 @@ aliases = [ "node/785" ]
 
 The ReactOS Project is seeking qualified individuals to fill multiple positions. Contracted **full time, part time, and scholarship (internship)** level positions are all available.
 
+If you do qualify for a position, you'll be joining the very small and exclusive ranks of people who know how to do NT systems development. And believe us, those ranks are very, very small. And yes, you will be gaining an immense amount of knowledge working with such people. 
 
 Available Positions
 ===
 
 Developer
 ---
-Some text about developer qualifications
+We are looking for people proficient in C and its a big plus if you are proficient in C++ too.C++ proficiency does not equate C proficiency so do not assume you have the latter if you have the former. Also keep in mind, again, that these are the minimum requirements. On top of these you need to be a good problem solver, belief in the mission of ReactOS, and most importantly, be able to demonstrate that you can work with the ReactOS codebase via a few patches.
```

Comment:
Nit: space after `.`.
Furthermore: explaining a difference between c and c++ is good, but for some positions c++ is more important,
other c is just fine.
(f.e. drivers / kernel mode stuff etc are generally c code, the shell and usermode applications lean more toward c++)

Reply:
Okay, I will be elaborating a bit about the C and C++ requirement part.

Reply:
I've made changes to the developer section. Please do have a look at it.